api/src/test/java/io/opencensus/trace/export/SpanDataTest.java

```diff
@@ -68,11 +68,21 @@
           TraceId.generateRandomId(random), SpanId.generateRandomId(random), TraceOptions.DEFAULT);
   private final SpanId parentSpanId = SpanId.generateRandomId(random);
   private final Map<String, AttributeValue> attributesMap = new HashMap<String, AttributeValue>();
+
+  // TODO: Decide whether to use a different class instead of LinkedList.
```

Comment:
In tests is not a problem. Please use what is suggested.

Reply:
Okay, this test was using List for each of the field types, so I just replaced LinkedList with ArrayList.